midealocal/device.py

```diff
@@ -246,19 +247,10 @@ def send_message_v2(self, data: bytes, query: bool = False) -> None:
             # raise exception to main loop
             raise SocketException
         try:
-            _LOGGER.debug(
```

Comment:
thank your very much for your review comments.
it just used to confirm the socket data has been matched and also send pass.
as we have a device status refresh delay issue. I'm not sure with the send datetime, and simply add two lines debug log before send and after send to confirm with it with v4.0 PR.
in fact, it always can match and if error matched, it will be catch by the try/except in main loop.
in addition, there's still a heartbeat packet every 10 seconds. it also will flood some logs lines.
so remove these teo lines is safe